src/main/java/de/chojo/repbot/commands/Info.java

```diff
@@ -0,0 +1,127 @@
+    }
+
+    public static Info create(Localizer localizer, Configuration configuration) {
+        var version = "undefined";
+        try (var in = Info.class.getClassLoader().getResourceAsStream("version")) {
+            version = new String(in.readAllBytes());
+        } catch (IOException e) {
+            log.error("Could not determine version.");
+        }
+        return new Info(localizer, version, configuration);
+    }
+
+
+    @Override
+    public boolean onCommand(MessageEventWrapper eventWrapper, CommandContext context) {
+        eventWrapper.reply(getResponse(eventWrapper))
+                .queue();
+        return true;
+    }
+
+    @NotNull
+    private MessageEmbed getResponse(MessageEventWrapper eventWrapper) {
+        if (contributors == null || lastFetch.isBefore(Instant.now().minus(5, ChronoUnit.MINUTES))) {
+            var request = HttpRequest.newBuilder().GET()
+                    .uri(URI.create("https://api.github.com/repos/rainbowdashlabs/reputation-bot/contributors?anon=1"))
+                    .header("accept", "application/vnd.github.v3+json")
+                    .header("User-Agent", "reputation-bot")
+                    .build();
+
+            try {
+                var response = client.send(request, HttpResponse.BodyHandlers.ofString());
+                List<GithubProfile> contributorProfiles = mapper.readerForListOf(GithubProfile.class).readValue(response.body());
+                contributors = contributorProfiles.stream().map(GithubProfile::toString).collect(Collectors.joining(", "));
+            } catch (IOException | InterruptedException e) {
+                log.error("Could not read response", e);
+            }
+            lastFetch = Instant.now();
+        }
+
+        return new LocalizedEmbedBuilder(localizer, eventWrapper)
+                .setTitle("command.info.embedTitle")
+                .setThumbnail(eventWrapper.getJda().getSelfUser().getEffectiveAvatarUrl())
+                .addField("command.info.contributor", contributors, false)
+                .addField("command.info.art", ART, true)
+                .addField("command.info.source", SOURCE, true)
+                .addField("command.info.version", version, true)
+                .addField("", "**"+localizer.localize("command.info.links", eventWrapper.getGuild(),
+                        Replacement.create("INVITE", configuration.links().invite()),
+                        Replacement.create("SUPPORT", configuration.links().support()),
+                        Replacement.create("TOS", configuration.links().tos()),
+                        Replacement.create("WEBSITE", configuration.links().website())) + "**", false)
+                .setColor(Colors.Pastel.BLUE)
+                .build();
+    }
+
+    @Override
+    public void onSlashCommand(SlashCommandEvent event) {
+        var eventWrapper = MessageEventWrapper.create(event);
+        event.reply(wrap(getResponse(eventWrapper))).queue();
```

Comment:
there is actually replyEmbeds or something like that now

Reply:
So much nicer :D